plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -145,6 +160,180 @@ public static function get_featured() {
 
 		wp_send_json( $featured );
 	}
+
+	public static function check_subscriptions( $screen ) {
+		$user_id = get_current_user_id();
+		if ( ! $user_id ) {
+			return;
+		}
```

Comment:
Maybe return early if the site is not connected.

Reply:
Correct. Will update it.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/47697/commits/76cc0415010edf43b50b94897ca4fc9bf34543d6.